Common/Servers/ReceiversBoss/src/RecvBossCore_mc.i

```diff
@@ -1,10 +1,10 @@
 //#define RB_DEBUG
 
-#define RECV_ADDRESS "192.167.189.2"
+#define RECV_ADDRESS "192.168.51.2"
 #define RECV_PORT 2096
-#define FS_ADDRESS "192.167.189.62"
+#define FS_ADDRESS "192.168.51.14"
 #define FS_PORT 5002
-#define SXKL_ADDRESS "192.167.189.2"
```

Comment:
This code seems to be part of the Common branch, are we sure we want to keep some IP addresses in this part of the repository? This code might have to run in multiple stations

Reply:
A switch at compile time avoids this piece of code is compiled for stations different from Medicina. 
Of course this should be included in a dedicated component.